src/main/scala/part4integrations/IntegratingAkka.scala

```diff
@@ -49,7 +49,7 @@ object IntegratingAkka {
 
 object ReceiverSystem {
   implicit val actorSystem = ActorSystem("ReceiverSystem", ConfigFactory.load("akkaconfig/remoteActors").getConfig("remoteSystem"))
-  implicit val actorMaterializer = ActorMaterializer()
+  implicit val actorMaterializer = Materializer(actorSystem)
 
```

Comment:
Symbol ActorMaterializer is deprecated. In recent versions of Akka, Materializer.apply(system) is the recommended way to create a system-wide materializer.

Reply:
👍